### PR TITLE
Updating breaking changes, as per 3.7

### PR DESCRIPTION
**Pull request by sfshaza2**
Routine cleanup

### CODE REVIEW

**Review by parlough — APPROVED** [review]
Looks good to me after a few minor fixes I pushed. I manually verified all these commits are included in 3.7.0.

**Comment by sfshaza2** [comment]
Yeah, I verified them, too. But thanks! It makes sense to double check.

**Comment by sfshaza2** [comment]
(I just wonder if we're missing any...)

File: src/release/breaking-changes/trackpad-gestures.md [hunk 1]
```diff
@@ -303,16 +303,16 @@ class Foo extends StatelessWidget {
 ```
 
 Please note: Use of raw `Listener` in this way could cause conflicts with other
```
Comment (atsansone) [inline src/release/breaking-changes/trackpad-gestures.md:305]: **chore:** Per [GSG](https://developers.google.com/style/tone#politeness-and-use-of-please), remove `Please`.

**question:** Should this be in a note block?

File: src/release/breaking-changes/trackpad-gestures.md [hunk 2]
```diff
@@ -303,16 +303,16 @@ class Foo extends StatelessWidget {
...
 
 #### Using `GestureDetector`
 
-If using Flutter 3.3.0, `RawGestureDetector` can be used instead of `GestureDetector`
+If using Flutter 3.3.0, `RawGestureDetector` could be used instead of `GestureDetector`
 to ensure each `GestureRecognizer` created by the `GestureDetector` has
 `supportedDevices` set to exclude `PointerDeviceKind.trackpad`. Starting in
-version 3.4.0, there will be a `supportedDevices` parameter directly on
+version 3.4.0, there is a `supportedDevices` parameter directly on
 `GestureDetector`.
```
Comment (atsansone) [inline src/release/breaking-changes/trackpad-gestures.md:316]: **suggestion:** Change this to active voice?
```suggestion
version 3.4.0, `GestureDetector` added a `supportedDevices` parameter.
```

File: src/release/breaking-changes/trackpad-gestures.md [hunk 3]
```diff
@@ -414,7 +414,8 @@ class Foo extends StatelessWidget {
 
 #### Using `RawGestureRecognizer`
 
-Explicitly ensure that `supportedDevices` does not include `PointerDeviceKind.trackpad`.
+Explicitly ensure that `supportedDevices`
+doesn't include `PointerDeviceKind.trackpad`.
```
Comment (atsansone) [inline src/release/breaking-changes/trackpad-gestures.md:418]: **question:** Why does this need to be `explicitly`? Can the developer add the `.trackpad` option without Flutter returning an error?

File: src/release/breaking-changes/trackpad-gestures.md [hunk 4]
```diff
@@ -494,9 +495,9 @@ class Foo extends StatelessWidget {
 #### Using `GestureRecognizer` and `Listener`
 
 After upgrading to Flutter 3.3.0, there won't be a change in behavior, as
-`addPointerPanZoom` needs to be called on each `GestureRecognizer` to allow
-it to track gestures. The following code will not receive pan gesture callbacks
-when the trackpad is scrolled.
+`addPointerPanZoom` must be called on each `GestureRecognizer` to allow
```
Comment (atsansone) [inline src/release/breaking-changes/trackpad-gestures.md:498]: **suggestion:** Imperative tone may read better here.
```suggestion
To allow `GestureRecognizer` to track gestures, call `addPointerPanZoom` on each `GestureRecognizer` method.
```

**Review by atsansone — COMMENTED** [review]
@sfshaza2 : A few suggestions and questions. Nothing blocking.

**Pull request merged**